addons/osio-addon/src/main/java/io/fabric8/launcher/osio/steps/booster/GolangBoosterUtility.java

```diff
@@ -0,0 +1,83 @@
+package io.fabric8.launcher.osio.steps.booster;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Utility class used to assist with customizing golang booster files
+ *
+ */
+public class GolangBoosterUtility {
```

Comment:
Not sure why this class is necessary, there are plenty of utility functions in commons-io 
 that does the same thing (and it's already available in launcher), like `org.apache.commons.io.FilenameUtils#getExtension`, `org.apache.commons.io.FileUtils#readFileToString(java.io.File, java.nio.charset.Charset)`, among others

Reply:
This class was meant to just hold helper methods.  I can definitely switch to using getExtension.